### PR TITLE
fix: users should be able to copy text from select inputs

### PR DESCRIPTION
**Pull request by FBalint**

### CODE REVIEW

File: src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}: {
+  value: StyledSelectProps["value"]
+  pathToCopy: string
+} & IndicatorsContainerProps<unknown, boolean, GroupBase<unknown>>) => {
+  const [copied, setCopied] = useState(false)
+  const successIconColor = useColorModeValue("green.800", "green.200")
+
+  const handleCopyToClipboard = async () => {
+    try {
+      await navigator.clipboard.writeText(parseFromObject(value, pathToCopy))
```
Comment (BrickheadJohnny) [inline src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx:26]: Is it intentional that you haven't used the `useClipboard` hook here?

File: src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      }, 1000)
+    } catch (err) {
+      console.error("Failed to copy:", err)
+    }
+  }
+
+  return (
+    <components.IndicatorsContainer {...rest}>
+      {value && (
+        <MotionIconButton
```
Comment (BrickheadJohnny) [inline src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx:39]: Do we really need a motion component here? 👀

File: src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx (hunk 1 above)
Comment (FBalint) [reply src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx:26]: Wow I missed this, thanks!

File: src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          backgroundColor={
+            hasCopied && transparentize(successIconColor, 0.12)(theme)
+          }
+          color={hasCopied && successIconColor}
+          icon={hasCopied ? <Check /> : <Copy />}
+          aria-label={"Copy"}
+          size={"xs"}
+          rounded={"full"}
+          variant={"ghost"}
+          onMouseDown={() => {
```
Comment (BrickheadJohnny) [inline src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx:33]: maybe `onClick` + `e.stopPropagation()` would work properly and then we wouldn't need to use `onMouseDown` here ;) Please check and lmk!

Comment (FBalint) [reply src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx:33]: I found that react-select uses `onMouseDown` to set the dropdown state, which happens before `onClick`. So we kind of need to get ahead of it by using onMouseDown here as well

Comment (BrickheadJohnny) [reply src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx:33]: Hmm, I see, thanks for the explanation.
The problem with the current solution is that it won't work on mobile (or at least I couldn't really click on the copy button), so let's brainstorm about this issue on Discord ;)

**Review by BrickheadJohnny — APPROVED** [review]

**Pull request merged**